naff/models/discord/channel.py

```diff
@@ -1122,12 +1129,7 @@ async def edit_permission(self, overwrite: PermissionOverwrite, reason: Optional
             reason: The reason for this change
         """
         await self._client.http.edit_channel_permission(
-            self.id,
-            overwrite_id=overwrite.id,
-            allow=overwrite.allow,
-            deny=overwrite.deny,
-            perm_type=overwrite.type,
-            reason=reason,
+            self.id, overwrite.id, overwrite.allow, overwrite.deny, overwrite.type, reason
```

Comment:
Why remove the parameter names here?

Reply:
Looks like a bleed over from another commit. I must not have created this branch cleanly like I thought I had. ill resolve